rpcserver.go

```diff
@@ -2661,8 +2661,9 @@ func (state *gbtWorkState) blockTemplateResult(bm *blockManager,
 		}
 
 		// Serialize the transaction for later conversion to hex.
-		txBuf := bytes.NewBuffer(make([]byte, 0, tx.SerializeSize()))
-		if err := tx.Serialize(txBuf); err != nil {
+		var txBuf bytes.Buffer
+		txBuf.Grow(tx.SerializeSize())
```

Comment:
None of these style changes are needed.  The existing code already allocates the buffer and passes it in.